hack/run-test-e2e.sh

```diff
@@ -64,7 +64,10 @@ export E2E_TOPOLOGY_MANAGER_POLICY="${E2E_TOPOLOGY_MANAGER_POLICY:-SingleNUMANod
 # --flake-attempts: rerun the test if it fails
 # -requireSuite: fail if tests are not executed because of missing suite
 ${BIN_DIR}/e2e-nrop-rte.test ${NO_COLOR} --ginkgo.v --ginkgo.timeout=5h --ginkgo.fail-fast --ginkgo.flake-attempts=2 --ginkgo.junit-report=${REPORT_DIR}/e2e.xml --ginkgo.skip='\[Disruptive\]|\[StateDirectories\]|\[NodeRefresh\]|\[local\]'
```

Comment:
We can annotate the bash script with `set -e` if we expect nothing to fail, instead of checking the exit code for each test

Reply:
Yes, probably better to switch to `-e`. I was looking for a targeted fix but it seems we need a deep rework here.

Reply:
yes